android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java

```diff
@@ -133,10 +133,16 @@ public void open(Context context, final ReadableMap options, final Promise promi
     setColor(builder, options, KEY_NAVIGATION_BAR_COLOR, "setNavigationBarColor", "navigation bar");
     setColor(builder, options, KEY_NAVIGATION_BAR_DIVIDER_COLOR, "setNavigationBarDividerColor", "navigation bar divider");
 
-    if (options.hasKey(KEY_DEFAULT_SHARE_MENU_ITEM) && 
-        options.getBoolean(KEY_DEFAULT_SHARE_MENU_ITEM)) {
-      builder.addDefaultShareMenuItem();
+    if(options.hasKey(KEY_DEFAULT_SHARE_MENU_ITEM)){
+      if(options.getBoolean(KEY_DEFAULT_SHARE_MENU_ITEM)){
+        builder.setShareState(CustomTabsIntent.SHARE_STATE_ON);
+      }else{
```

Comment:
Hey mate, can you fix indentation here please?

Reply:
@jdnichollsc Done . Pls check